webapp.js

```diff
@@ -49,12 +48,7 @@ app.use(express.static('dist'));
 app.use(bodyParser.json());
 
 app.get('/api/data', function(req, res) {
```

Comment:
@achiu8 ok got it. I tried moving it up front, but still not working. Would this require a promise or something?